apps/mail/components/create/ai-chat.tsx

```
@@ -320,28 +285,23 @@ export function AIChat({
-                      <ToolResponse
-                        key={part.toolInvocation.toolName}
-                        toolName={part.toolInvocation.toolName}
-                        result={part.toolInvocation.result}
-                        args={part.toolInvocation.args}
-                      />
-                    ) : null,
+                <div key={`${message.id}-${index}`} className="mb-2 flex flex-col">
+                  {toolParts.map(
+                    (part, index) =>
+                      part.toolInvocation &&
+                      part.toolInvocation.result && (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)